salt/transport/tcp.py

```diff
@@ -664,14 +871,15 @@ def _stream_return(self):
                     else:
                         if self._on_recv is not None:
                             self.io_loop.spawn_callback(self._on_recv, header, body)
+                            # await self._on_recv(header, body)
```

Comment:
```suggestion
```

Reply:
This is in the deprecated MessageClient class and can be ignored.